src/analysis/raster/qgskde.h

```diff
@@ -39,7 +39,7 @@ class ANALYSIS_EXPORT QgsKernelDensityEstimation
 {
   public:
     //! Kernel shape type
-    enum KernelShape
+    enum class KernelShape : int
     {
       KernelQuartic = 0,  //!< Quartic kernel
```

Comment:
```suggestion
      Quartic SIP_MONKEYPATCH_COMPAT_NAME( KernelQuartic )= 0 ,  //!< Quartic kernel
```
elsewhere here too -- eg for OutputValues we should remove the "Output" prefix.

Reply:
Should be fixed